src/assignment/assignment.controller.ts

```diff
@@ -88,8 +87,8 @@ export class AssignmentController {
   }
 
   // Delete a Assignment
-  @Delete('/delete')
-  async deleteAssignment(@Res() res, @Query('uid') uid) {
+  @Delete('/:uid')
```

Comment:
Why you have named it uid?
name it something like AssignmentId or so.

Reply:
Ok ,will change this